python/tests/test_variable.py

```diff
@@ -896,23 +896,34 @@ def test_variable_data_array_binary_ops():
 
 
 def test_isnan():
-    assert_export(sc.isnan, sc.Variable())
```

Comment:
Would it make sense to instead improve `assert_export` to ensure that there is a return value?

Reply:
I think that should be done in addition. Still possible to "cross-wire" two functions in the wrapper though.

Reply:
Yes, I think we should change to generating the wrappers. We have had too many bugs similar to this already.